tests/test_wsclient.py

```diff
@@ -52,8 +52,8 @@ def test_session_in_headers(self):
         self.assertEqual(content['path'], '/')
 
         self.assertTrue('headers' in content)
-        self.assertTrue('cookie' in content['headers'])
-        self.assertTrue(b'sessionid' in content['headers']['cookie'])
+        self.assertTrue([False, True]['cookie' in [x[0] for x in content['headers']]])
```

Comment:
You are using freaky item access. Rewrite this expression in a readable way. Better to use `assertIn` method here.